storagenode/storagenodedb/database.go

```diff
@@ -16,7 +16,7 @@ import (
 	_ "github.com/mattn/go-sqlite3" // used indirectly
 	"github.com/zeebo/errs"
 	"go.uber.org/zap"
-	monkit "gopkg.in/spacemonkeygo/monkit.v2"
+	"gopkg.in/spacemonkeygo/monkit.v2"
```

Comment:
@egonelbre are we still trying to use the monkit package name alias? I know when I autosave + gofmt in Goland it removes the alias because it says it's redundant.

Reply:
The issue is that the path contains "monkit.v2", module name is `monkit.v2` and package name `monkit`.

Goland seems to take the easy approach by ignoring versions at the end of the import. However, goimports uses the rule that when import path (ignoring module versioning) and package name differ, then it uses an alias to keep things clear.

If `goimports -w -local storj.io` formats the code the same way, then I'm fine with either way. If it's not, then let's prefer `goimports`, because that's the default for VS Code.